teams.md/src/pages/templates/migrations/botbuilder/turn-context/proactive-activities.mdx

```diff
@@ -11,4 +11,9 @@ import TabItem from '@theme/TabItem';
 
 # Proactive Activities
 
-<LanguageInclude section="content" />
+The BotBuilder proactive message flow requires storing a conversation reference. 
+In Teams SDK, we expose a <LanguageInclude section="method-name" /> method in the `App` class, almost identical to the one 
```

Comment:
<img width="869" height="187" alt="Image" src="https://github.com/user-attachments/assets/71e4d102-413c-4148-8421-f87e9f3a7e32" />

This thing shows up in all the languages.

Reply:
Notice the "LanguageInclude..."